packages/app/features/fetch_after_connection.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Fetch after connection
+
+  After a repo is connected, we automatically run a fetch in the
+  background.
+
+  Scenario: Succesful fetch after a new connection
+    Given a remote repo with commits on the "main" branch
+    When a consumer connects the remote repo
+    Then the repo should be fetched
```

Comment:
looks like your IDE is not properly configured (?)

Reply:
In what way? Whitespace?

Reply:
It's not adding a newline character at the end of the last line.

Reply:
Seems to be in feature files only